src/SearchFilters.js

```diff
@@ -35,29 +35,53 @@ export let FiltersBox = withTheme(({ theme: { Box }, ...props }) => (
 ))
 FiltersBox.displayName = 'FiltersBox'
 
-let BasicSearchFilters = ({ setMode, trees, children, BasicFilters }) => (
-  <div>
-    <Flex alignItems="center" justifyContent="space-between">
-      <Flex alignItems="center">
-        <h1>Filters</h1>
-        <ToggleFiltersButton onClick={() => setMode('resultsOnly')} />
-      </Flex>
-      <div>
-        <TreePauseButton children={children} />
-      </div>
-    </Flex>
-    <LabelledList list={trees} Component={BasicFilters} />
-    <LinkButton onClick={() => setMode('builder')} style={{ marginTop: 15 }}>
-      Switch to Advanced Search Builder
-    </LinkButton>
+let PopOverItem = ({ children, ...props }) => (
+  <div style={{ margin: 10, cursor: 'pointer' }} {...props}>
+    {children}
   </div>
 )
 
+let BasicSearchFilters = ({ setMode, trees, children, BasicFilters }) => {
+  let [isOpen, setIsOpen] = React.useState(false)
+  return (
+    <div>
+      <Flex alignItems="center" justifyContent="space-between">
+        <Flex alignItems="center">
+          <h1>Filters</h1>
+        </Flex>
+        <div>
+          <Popover
+            isOpen={isOpen}
+            onClose={() => setIsOpen(false)}
+            style={{ width: 180, right: 0, top: 40 }}
```

Comment:
If we use DropdownItem, I don't think we'll need any styling. In general, we should defer most styling in contexture-react core to theme components

Reply:
No styling on the popover looks super weird:

![image](https://user-images.githubusercontent.com/910303/84942189-4d339d80-b097-11ea-9da4-cf3aaff9466d.png)

Reply:
Removed the styling ... It looks weird but I guess its ok.